README.md

```diff
@@ -4,3 +4,46 @@ This is a starter kit to help you build your own map for [WorkAdventure](https:/
 
 To understand how to use this starter kit, follow the tutorial at [https://workadventu.re/map-building](https://workadventu.re/map-building).
 
+## Introduction
+
+- WorkAdventure is a open source virtual office solution
+- Offers spontaneous meetings without the hassle of setting a date
+- Maps can be designed and structured as pleased
+
+## Submitting a Map-Change
+
+- Open issue with suggetion for change
+- No one complains -> submit pull request with changes
+- Important! Concurrently working on the same map is very problematic
+  - Therefore, changes should be as small as possible and quickly transition from issue to PR
+
+## Working with Tiled
+
+Tiled: Intuitive tool for creating tilemaps
+
+### Tilesets
+  - Atlasse used as a register for tiles to draw
+  - For workadventure, tiles should be 32 x 32 pixel
+  - Add a tileset via import of a png
+  - Important! Tilesets have to be embedded into the tilemap JSONs
+
+### Tilemap
+  - A map in which tiles from one or more tilesets are placed
+  - Multiple layers
+  - Important! Has to be exported as JSON
+
+### Advanced
+  - Animated Tiles
+  - TerrainSets
+
+### WorkAdventure Integration
```

Comment:
I'd directly add an introductory sentence to this section with a link to the map-building tutorial. People should see/use that link first. What we have here is just a summary/teaser.